src/components/views/settings/tabs/room/RolesRoomSettingsTab.js

```diff
@@ -151,6 +151,22 @@ export default class RolesRoomSettingsTab extends React.Component {
         client.sendStateEvent(this.props.roomId, "m.room.power_levels", plContent);
     };
 
+    _onUserPowerLevelChanged = (value, powerLevelKey) => {
+        const client = MatrixClientPeg.get();
+        const room = client.getRoom(this.props.roomId);
+        const plEvent = room.currentState.getStateEvents('m.room.power_levels', '');
+        let plContent = plEvent ? (plEvent.getContent() || {}) : {};
+
+        // Clone the power levels just in case
+        plContent = Object.assign({}, plContent);
+
+        // powerLevelKey should be a user ID
+        if (!plContent['users']) plContent['users'] = {};
+        plContent['users'][powerLevelKey] = value;
+
+        client.sendStateEvent(this.props.roomId, "m.room.power_levels", plContent);
```

Comment:
should we handle any send errors here? Show a dialog on fail?

Reply:
I suppose that'd be responsible of us... Will add